### ISSUE TITLE
`WPS344` ignores `//= 0`

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#974]
# Bug report

## What's wrong

This code is not marked as invalid:

```python
x //= 0
```

## How is that should be

It should raise `WPS344`. See here: https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/visitors/ast/operators.py#L100

We also need to check for `ast.FloorDiv` as well.
And edit a test here: https://github.com/wemake-services/wemake-python-styleguide/blob/master/tests/test_visitors/test_ast/test_operators/test_useless_math.py#L46

### flake8 information

Contents of `flake8 --bug-report`:

### pip information

Contents of `pip freeze`:
